### ISSUE TITLE
Scaling depth map to real-world metrics

### ISSUE THREAD

**Issue by imneonizer** [ibaiGorordo/ONNX-GLPDepth#1]
Can you provides some instructions on how to convert the final depth map output to real-world metrics like `meters`
For example I detect a car in the original frame and the using the depth map is it possible to tell how far the car is ?

I found this repository https://github.com/utiasSTARS/learned_scale_recovery which seems to address the same problem but is a bit complicated to setup.

**Comment by ibaiGorordo** [comment]
Yeah, that is a complicated problem. I have not looked at that repository, but based on the image, they seem to be something easy. Since they know the height and orientation of the camera, they know with trigonometry the distance to the ground (only when the ground is flat).

If you compare the expected actual distance at certain points where the camera is looking at the road, with the estimated relative depth, you can probably use linear regression to estimate the scale. If I remember correctly, this model was quite stable, so you probably don't need to even recalculate the scale in every frame.
